components/formats-gpl/src/loci/formats/in/FlexReader.java

```diff
@@ -814,9 +813,13 @@ private void parseFlexFile(int currentWell, int wellRow, int wellCol,
       ifd = file.ifds.get(0);
     }
     else {
-      TiffParser parser = new TiffParser(file.file);
-      ifd = parser.getFirstIFD();
-      parser.getStream().close();
+        RandomAccessInputStream ras = new RandomAccessInputStream(file.file);
+        try {
+            TiffParser parser = new TiffParser(ras);
+            ifd = parser.getFirstIFD();
+        } finally {
+            if (ras != null) ras.close();
```

Comment:
Few things:
 * It's impossible for `ras` to be `null` here.
 * Whitespace is off (4 rather than 2 spaces)
 * Commit message may be misleading. Why was this change made?